ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -19,9 +19,15 @@ const LargeText = styled.div`
   font-size: 1.2em;
 `
 
-const UcscBrowserLink = ({ variant, useLiftover, includeEnd }) => {
-  const chrom = useLiftover ? variant.liftedOverChrom : variant.chrom
-  const pos = parseInt(useLiftover ? variant.liftedOverPos : variant.pos, 10)
+const TRANSLOCATION_SV = 'BND'
+
+const UcscBrowserLink = ({ variant, useLiftover, includeEnd, isEnd }) => {
+  const variantChrom = isEnd ? variant.endChrom : variant.chrom
+  const variantPos = isEnd ? variant.end : variant.pos
+  const liftedVariantChrom = isEnd ? variant.rg37LocusEnd?.contig : variant.liftedOverChrom
+  const liftedVariantPos = isEnd ? variant.rg37LocusEnd?.position : variant.liftedOverPos
+  const chrom = useLiftover ? liftedVariantChrom : variantChrom
+  const pos = parseInt(useLiftover ? liftedVariantPos : variantPos, 10)
```

Comment:
you end up assigning a lot of variables here, many of which don't get used. I think it would be cleaner to use `let` variables and if/else blocks here. while the code is much longer, its much more explicit to the reader about how the final variablew are getting set
```
let chrom = variant.chrom
let rawPos = variant.pos
if (useLiftover) {
  if (isEnd && rg37LocusEnd) {
    chrom = variant.rg37LocusEnd.contig
    pos = variant.rg37LocusEnd.position
  } else {
    chrom = variant.liftedOverChrom
    rawPos = variant.liftedOverPos
  }
} else if (isEnd) {
  chrom = variant.endChrom || chrom
  rawPos = variant.end
}
const pos = parseInt(rawPos, 10)
```